src/commands/uplink/NewStatus.ts

```diff
@@ -0,0 +1,205 @@
+import Command from '../../Command.js';
+import CommandBinaryBuffer, {IBatteryVoltage} from '../../CommandBinaryBuffer.js';
+import * as hardwareTypes from '../../constants/hardwareTypes.js';
+import roundNumber from '../../utils/roundNumber.js';
+
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface IStatusBase {}
+
+interface IProduct {
+    version: number
+    type: number
+}
+
+
+interface IGasStatus extends IStatusBase {
+    voltage: IBatteryVoltage
+
+    /**
+     * battery internal resistance, in mΩ
+     *
+     * 65535 === undefined
+     */
+    internalResistance: number | undefined
+
+    temperature: number
+
+    /**
+     * Remaining battery capacity in percents.
+     * undefined - unknown capacity
+     *
+     */
+    remindedBatteryCapacity: number | undefined
+
+    lastEventSequenceNumber: number
+}
+
+
+const COMMAND_ID = 0x14;
+const COMMAND_TITLE = 'NEW_STATUS';
+const COMMAND_BODY_MAX_SIZE = 20;
+const UNKNOWN_RESISTANT = 65535;
+
+// max battery capacity, 254 - 100%
+const UNKNOWN_BATTERY_CAPACITY = 255;
+
+
+/**
+ * NewStatus command parameters
+ */
+interface INewStatusParameters {
```

Comment:
let's put all interfaces together

Reply:
reworked in all files, utils/message.ts and utils/getHexFromBytes.ts were already structured in this way